debian/changelog

```diff
@@ -16,7 +24,7 @@ python-ovh (0.4.4) trusty; urgency=medium
   * Add debian folder
   * fix: remove Python SNI warnings, OVH API does not need SNI (#35)
   * fix: coveralls version
-  * [auto] bump version to v0.4.6
+  * [auto] bump version to v0.4.7
```

Comment:
Argh, I completely forgot about the packaging... I should probably remove it altogether. Fixing